content/docs/latest/spiffe-about/get-involved.md

```diff
@@ -36,5 +36,5 @@ Day to day contributions are vetted by the project's maintainers. Overall projec
 
 Find SPIFFE and SPIRE logos here:
 
-* [SPIFFE](https://branding.cncf.io/projects/spiffe)
-* [SPIRE](https://branding.cncf.io/projects/spire)
+* [SPIFFE](https://branding.cncf.io/tree/master/projects/spiffe)
+* [SPIRE](https://branding.cncf.io/tree/master/projects/spire)
```

Comment:
Per the README in [cncf/cncf-branding](https://github.com/cncf/cncf-branding#readme) this site has been retired, and now blindly redirects to [cncf/artwork](https://github.com/cncf/artwork). As a result, the paths to the artwork folders need to be "github paths". We could go directly to the github links and cut out the redirect if we wanted? Happy to change this if that is preferred.

Reply:
I'd rather go with the github links directly, not that I'm concerned about an extra redirect, but would be better in case that the cncf removes the redirects in the future